include/ginkgo/core/base/array.hpp

```diff
@@ -354,6 +380,8 @@ class Array {
 
     /**
      * Resizes the array so it is able to hold the specified number of elements.
+     * To a view, this only resets the pointer. Afterwards, the  only way to use
```

Comment:
nit: a space too much
Also, I would clarify it a bit more:
```suggestion
     * In a view, this resets the pointer, so it does not point to anything
     * (num_elems will also be set to 0).
```